src/lib/components/header.svelte

```diff
@@ -119,3 +155,63 @@
 		</Drawer.Root>
 	</nav>
 </header>
+
+<Dialog.Root bind:open={isSearchOpen}>
+	<Dialog.Content class="p-0 sm:max-w-[600px] [&>button]:hidden">
```

Comment:
`p-0 sm:max-w-[600px] [&>button]:hidden` See if removing `p-0 [&>button]:hidden` makes any difference. If not then remove them